.github/workflows/main.yml

```diff
@@ -21,13 +21,19 @@ jobs:
         # found ("dyld: Library not loaded; '@rpath/libopenvino.2310.dylib'"). See
         # https://github.com/abrown/openvino-rs/actions/runs/6423141936/job/17441022932#step:7:154
         os: [ubuntu-20.04, ubuntu-22.04, windows-latest]
-        version: [2022.3.0, 2023.1.0, 2023.2.0, 2024.0.0]
+        version: [2023.2.0, 2024.0.0, 2024.1.0]
         apt: [false]
         # We also spot-check that things work when installing from APT by adding to the matrix: see
         # https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#expanding-or-adding-matrix-configurations
+        # APT install and check oldest supported version 2023.2.0
         include:
           - os: ubuntu-22.04
-            version: 2024.0.0
+            version: 2023.2.0
```

Comment:
I meant that we could try `2022.3.0` here if we want to show that even older versions still work on Linux.